.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI for React App
+
+on:
+  push:
+    branches: [ test/testing-jest-features ]
```

Comment:
Is it possible to rather target opened merge request events? Ideally, a pre-build merge and run of the tests and linting should be good.

Reply:
I believe it should, but I will double-check. But on the same branch or develop since that's where we have PR's at the moment?